ballista/scheduler/scheduler_config_spec.toml

```diff
@@ -51,7 +51,6 @@ doc = "etcd urls for use when discovery mode is `etcd`. Default: localhost:2379"
 default = "std::string::String::from(\"localhost:2379\")"
 
 [[param]]
-abbr = "h"
```

Comment:
this parameter is reserved and always implemented by configure_me.

so, remove is right. 👍